package.json

```diff
@@ -20,6 +20,7 @@
     "axios": "^0.27.2",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-bootstrap": "^2.4.0",
```

Comment:
On line 7 just below the test command

```
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
"start": "parcel src/index.html"
  },

```

This way you can only run ```npm run start``` or ```yarn start``` depending on the one you're using.